lib/bugsnag.js

```diff
@@ -91,7 +91,9 @@ Bugsnag.notify = function(error, options, cb) {
     }
     if (!Bugsnag.shouldNotify()) {
         if (cb) {
-            cb();
+            cb(new Error(!Configuration.apiKey ?
```

Comment:
I'm thinking a simple `else` would do the trick, as the call to `Bugsnag.shouldNotify()`, a line or two above, has already confirmed it's false.